std/parallelism.d

```diff
@@ -3346,6 +3405,7 @@ private void submitAndExecute(
     alias PTask = typeof(scopedTask(doIt));
     import core.stdc.stdlib;
     import core.stdc.string : memcpy;
```

Comment:
This is not necessary because of the same import inside the foreach below, right?  Get rid of one of them.